### PR TITLE
fix: explore button on mobile

### LINKED ISSUE #4683: Explorer Mobile Tabs
<details><summary>But mobile is not very optimal yet</summary>

Screenshot 2023-01-13 at 15-10-52 Low minting fees and carbonless NFTs KodaDot - Kusama NFT Market Explorer

</details>

We can expand this in another issue

#### Note
- mobile design need some love

_Originally posted by @roiLeo in https://github.com/kodadot/nft-gallery/pull/4577#pullrequestreview-1247709939_

### PR DESCRIPTION
**Pull request by preschian**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).
👇 \_ Let's make a quick check before the contribution.

### PR type

- [x] Bugfix
- [ ] Feature
- [ ] Refactoring

### What's new?

- [x] PR closes #4683

### Before submitting Pull Request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [ ] I've posted a screenshot of demonstrated change in this PR

### Optional

- [ ] I've tested it at </rmrk/collection/26902bc2f7c20c546a-1FVG7>
- [ ] I've tested PR on mobile and everything seems works
- [ ] I found edge cases
- [ ] I've written some unit tests 🧪

### Had issue bounty label?

- [x] Fill up your KSM address: [Payout](https://beta.kodadot.xyz/transfer/?target=DY4SQF2iD456tH89aQtz5wv1EV3BbSW8wKKuMcwbmXaj1pM)

### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

### Screenshot

- [x] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

image

### CODE REVIEW

File: components/shared/exploreTabsFilterSort/ExploreTabsFilterSort.vue [hunk 1]
```diff
@@ -56,18 +57,20 @@ const updateTab = (val) => {
...
-.explore-heading {
-  font-weight: 700;
-}
+@import '@/styles/abstracts/variables';
 
 .explore-tabs {
-  .btn-items {
-    width: 240px;
-    border-left: none;
+  display: flex;
```
Comment (Jarsen136) [inline components/shared/exploreTabsFilterSort/ExploreTabsFilterSort.vue:63]: let's use`is-flex` instead

File: styles/global.scss [hunk 2]
```diff
@@ -180,3 +180,7 @@ body {
 hr {
   background: $k-grey;
 }
+
+.section {
+  padding: 3rem 1.25rem;
```
[the remark this answers is not in the document]
Comment (roiLeo) [reply styles/global.scss:185]: Yup [section-padding](https://bulma.io/documentation/layout/section/#variables) IMO should be put in `styles/abstracts/derived-variables`

File: components/shared/exploreTabsFilterSort/ExploreTabsFilterSort.vue [hunk 3]
```diff
@@ -1,22 +1,22 @@
 <template>
-  <div class="section pb-0">
-    <h1 class="title container">{{ $t('explore') }}</h1>
+  <div>
+    <h1 class="title">{{ $t('explore') }}</h1>
```
Comment (roiLeo) [inline components/shared/exploreTabsFilterSort/ExploreTabsFilterSort.vue:3]: I would put this in `explore-layout` layout

File: layouts/explore-layout.vue [hunk 4]
```diff
@@ -1,7 +1,11 @@
 <template>
   <div class="min-h-full is-flex is-flex-direction-column is-clipped">
     <Navbar />
-    <ExploreTabsFilterSort />
+    <div class="section pb-0">
+      <div class="container">
+        <ExploreTabsFilterSort />
+      </div>
+    </div>
```
Comment (roiLeo) [inline layouts/explore-layout.vue:8]: we can wrap `main` here, it will allow you not to writte `section.section div.container` two time

**Review by roiLeo — COMMENTED** [review]
small thing

File: styles/abstracts/_derived-variables.scss [hunk 5]
```diff
@@ -119,3 +118,5 @@ $family-primary: $family-monospace;
 $text-strong: $scheme-invert;
 
 $light-text: #f8f4f6;
+
+$section-padding: 3rem 1.25rem !default;
```
Comment (Jarsen136) [inline styles/abstracts/_derived-variables.scss:122]: Do we really need this `!default` 🤔

**Review by Jarsen136 — COMMENTED** [review]
small stuff, otherwise code lgtm

**Review by Jarsen136 — APPROVED** [review]

**Review by roiLeo — APPROVED** [review]
✅ Noice! wfm

**Comment by yangwao** [comment]
pay 50 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $50 @ 34.25 USD/KSM ~ 1.46 $KSM
🧗 [DY4SQF2iD456tH89aQtz5wv1EV3BbSW8wKKuMcwbmXaj1pM](https://kusama.subscan.io/account/DY4SQF2iD456tH89aQtz5wv1EV3BbSW8wKKuMcwbmXaj1pM)
🔗 [0x517e2bbc983ba3f7b9737a52a1ff6dcc1841425529b1a9ad957405ef182f8380](https://kusama.subscan.io/extrinsic/0x517e2bbc983ba3f7b9737a52a1ff6dcc1841425529b1a9ad957405ef182f8380)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $50 @ 35.11 USD/KSM ~ 1.424 $KSM
🧗 [DY4SQF2iD456tH89aQtz5wv1EV3BbSW8wKKuMcwbmXaj1pM](https://kusama.subscan.io/account/DY4SQF2iD456tH89aQtz5wv1EV3BbSW8wKKuMcwbmXaj1pM)
🔗 [0x0288bc7e2ccfe582a72a60244322aefbacf4c017a19513e308bb6b5a855c0e63](https://kusama.subscan.io/extrinsic/0x0288bc7e2ccfe582a72a60244322aefbacf4c017a19513e308bb6b5a855c0e63)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**